src/ManifestAsset.js

```diff
@@ -112,6 +113,25 @@ class ManifestAsset extends Asset {
         }
     }
 
+    processURLOverrides() {
+        const urlOverrides = this.ast.chrome_url_overrides
+        if (urlOverrides.bookmarks) {
+          urlOverrides.bookmarks = this.processSingleDependency(
```

Comment:
Sorry minor nitpick here, but could you please use prettier to autoformat the indentation here? (this project has a prettier config)

Reply:
Not minor at all, you're right :) I'm just used to my editor doing it for me, and it didn't in this case. I formatted it and did a push --force to avoid creating a new commit